announce.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"bytes"
+	"database/sql"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"text/template"
+	"time"
+)
+
+var cmdAnnounce = &Command{
+	UsageLine: "announce",
+	Short:     "Kündigt nächsten Stammtisch oder nächste c¼h an",
+	Long: `Announced den nächsten Stammtisch oder die nächste c¼h,
+je nachdem, was am nächsten Donnerstag ist.`,
+	Flag:         flag.NewFlagSet("announce", flag.ExitOnError),
+	NeedsDB:      true,
+	RegenWebsite: false,
+}
+
+var targetmailaddr = flag.String("announceAddress", "ccchd@ccchd.de", "Mailadresse, an die Ankündigungen gehen sollen.")
+
+func init() {
+	cmdAnnounce.Run = RunAnnounce
+}
+
+func isStammtisch(date time.Time) (stammt bool, err error) {
+	err = db.QueryRow("SELECT stammtisch FROM termine WHERE date = $1", date).Scan(&stammt)
+	return
```

Comment:
Entweder keine naked returns (falls es sie nicht auch woanders gibt hier, Konsistenz überwiegt), oder ein issue filen dass wir alle naked returns weg refactoren. Naked returns verstecken shadowing-bugs und waren ein Fehler in go 1.